src/cow-react/common/services/getAnchorCurrency/index.ts

```diff
@@ -0,0 +1,66 @@
+import { SupportedChainId } from 'constants/chains'
+import { Currency, CurrencyAmount, Token } from '@uniswap/sdk-core'
+import { DAI, USDC_MAINNET, USDT, WBTC, WRAPPED_NATIVE_CURRENCY } from 'constants/tokens'
+import { DAI_RINKEBY, USDT_RINKEBY, USDC_RINKEBY, WBTC_RINKEBY } from 'utils/rinkeby/constants'
+import { DAI_GOERLI, USDT_GOERLI, USDC_GOERLI, WBTC_GOERLI } from 'utils/goerli/constants'
+import { USDC_GNOSIS_CHAIN, USDT_GNOSIS_CHAIN, WBTC_GNOSIS_CHAIN, WXDAI } from 'utils/gnosis_chain/constants'
+import { NATIVE_CURRENCY_BUY_ADDRESS } from 'constants/index'
+
+const STABLE_COINS: { [key in SupportedChainId]: Token[] } = {
+  [SupportedChainId.MAINNET]: [USDC_MAINNET, USDT, DAI],
+  [SupportedChainId.GNOSIS_CHAIN]: [USDC_GNOSIS_CHAIN, USDT_GNOSIS_CHAIN, WXDAI],
+  [SupportedChainId.GOERLI]: [USDC_GOERLI, USDT_GOERLI, DAI_GOERLI],
+  [SupportedChainId.RINKEBY]: [USDC_RINKEBY, USDT_RINKEBY, DAI_RINKEBY],
+}
```

Comment:
Okay, this is basic, but would think is good enough!

Reply:
Maybe we could add a few more.

https://www.coingecko.com/en/categories/stablecoins

For future versions, we can evaluate coingecko as a source of truth to tell if its a stable coin. For now, your implementation is more than good enough

Reply:
Added TODO for the future